ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/fpe/Radix.java

```diff
@@ -14,24 +14,34 @@ public enum Radix {
   /**
    * Digits 0-9
    */
-  DIGITS(10, 6, 56, '0'),
+  DIGITS("0123456789", 6, 56, '0'),
 
   /**
    * lower case letters a-z
    */
-  LOWER(26, 5, 40, 'a'),
+  LOWER("abcdefghijklmnopqrstuvwxyz", 5, 40, 'a'),
 
   /**
    * Digits 0-9 and lower case letters a-z
    */
-  DIGITS_LOWER(36, 4, 36, 'a');
+  DIGITS_LOWER("0123456789abcdefghijklmnopqrstuvwxyz", 4, 36, 'a'),
 
-  private final int value;
+  /**
+   * lower case letters A-Z
```

Comment:
Comment should saw upper instead of lower

Reply:
good catch - updated